utilities/after_success.sh

```diff
@@ -10,6 +10,29 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
     mvn cobertura:cobertura coveralls:report
     mvn site-deploy -DskipTests=true --settings=target/travis/settings.xml
     mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
+
+    # Update "site/latest/index.html" to redirect to the newly generated website
+    git config --global user.name "Travis CI"
+    git config --global user.email ${CI_DEPLOY_USERNAME}@github.com
+    mkdir tmp_gh-pages # Avoid clashing filenames when cloning repo
+    cd tmp_gh-pages
+    git clone https://github.com/GoogleCloudPlatform/gcloud-java.git
```

Comment:
Travis already has `master` checked out for you, so you can just grab it from there.